models/elasticsearch.go

```diff
@@ -82,6 +82,8 @@ type ESSourceDocument struct {
 	Published       bool                `json:"published,omitempty"`
 	Language        string              `json:"language,omitempty"`
 	Survey          string              `json:"survey,omitempty"`
+	PopulationType  string              `json:"population_type,omitempty"`
```

Comment:
Could we change this to a type like the one defined here, please?
https://github.com/ONSdigital/dp-search-data-importer/blob/d5d831b47f074f7461f80f070b2976e3e6cb3be6/models/elastic.go#L63